Update README (#447)
wulkanowy/wulkanowy: README.en.md

[![Travis](https://img.shields.io/travis/com/wulkanowy/wulkanowy/master.svg?style=flat-square)](https://travis-ci.com/wulkanowy/wulkanowy)
[![Codecov](https://img.shields.io/codecov/c/github/wulkanowy/wulkanowy/master.svg?style=flat-square)](https://codecov.io/gh/wulkanowy/wulkanowy)
[![Discord](https://img.shields.io/discord/390889354199040011.svg?style=flat-square)](https://discord.gg/vccAQBr)
[![F-Droid](https://img.shields.io/f-droid/v/io.github.wulkanowy.svg)](https://f-droid.org/packages/io.github.wulkanowy/)
[![Last release](https://img.shields.io/github/release/wulkanowy/wulkanowy.svg?logo=github)](https://github.com/wulkanowy/wulkanowy/releases)

> [Polish version of README](README.md)

Reviewer: ```suggestion
[Polska wersja README](README.md)
```

# Wulkanowy

> Unofficial VULCAN UONET register application for student and parent

Reviewer: ```suggestion
Unofficial android VULCAN UONET+ register client for student and parent
```

## Features

* logging in using the email and password
* functions from the register website
    * grades
    * grade statistics
    * attendance
    * percentage of attendance
    * exams
    * timetable
    * completed lessons
    * messages
    * homework
    * notes
    * lucky number
* calculation of the average

> * notifications eg about a new grade

Reviewer: ```suggestion
* notifications, e.g. about a new grade
```

* dark and black (AMOLED) theme
* offline mode
* no ads

## Download

[Development versions](https://wulkanowy.github.io/#download)  

> [Beta version (Google Play)](https://play.google.com/store/apps/details?id=io.github.wulkanowy)

Reviewer: Maybe use google play badge here? https://play.google.com/intl/en_us/badges/
